forge-gui/res/cardsfolder/upcoming/invasion_of_ulgrotha_grandmother_ravi_sengir.txt

```diff
@@ -0,0 +1,23 @@
+Name:Invasion of Ulgrotha
+ManaCost:4 B
+Types:Battle Siege
+Defense:5
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDrain | TriggerDescription$ When CARDNAME enters the battlefield, it deals 3 damage to any other target and you gain 3 life.
+SVar:TrigDrain:DB$ DealDamage | ValidTgts$ Creature.Other,Player,Planeswalker.Other,Battle.Other | TgtPrompt$ Select any target | NumDmg$ 3 | SubAbility$ DBGainLife
+SVar:DBGainLife:DB$ GainLife | Defined$ You | LifeAmount$ 3
+DeckHas:Ability$Counters|LifeGain
```

Comment:
Why Counters?

Reply:
I think @tool4ever said something about the other side of double sided card overwriting the hints of the other side so I've bundled them together